File: PROGRAMMERS/lv0/javascript/중복된 숫자 개수.js
function solution(array, n) {
  return array.filter((i) => i === n).length;
}

let inp = [
  [[1, 1, 2, 3, 4, 5], 1],
  [[0, 2, 3, 4], 1],
];

for (let i of inp) {
  console.log(solution(i[0], i[1]));
}

File: PROGRAMMERS/lv0/python/중복된 숫자 개수.py
def solution(array, n):
    return len(list(filter(lambda x:x==n, array)))

inps = [
    [[1, 1, 2, 3, 4, 5], 1],
    [[0, 2, 3, 4], 1]
]

for (array, n) in inps:
    print(solution(array, n))